mailstream/phpmailer/class.phpmailer.php

```diff
@@ -3287,7 +3287,7 @@ protected function serverHostname()
         $result = 'localhost.localdomain';
         if (!empty($this->Hostname)) {
             $result = $this->Hostname;
-        } elseif (isset($_SERVER) and array_key_exists('SERVER_NAME', $_SERVER) and !empty($_SERVER['SERVER_NAME'])) {
+        } elseif (array_key_exists('SERVER_NAME', $_SERVER) and !empty($_SERVER['SERVER_NAME'])) {
```

Comment:
This can be simplified further.
```suggestion
        } elseif (!empty($_SERVER['SERVER_NAME'])) {
```

Reply:
The phpmailer is a vendor code and I tried to change it a little at possible. It is also possible that the `SERVER_NAME` key does not exists, e.g. in command line mode.

Reply:
I know, and it's fine, `empty()` takes care of missing array keys as well, just like `unset()`.